### PR TITLE
Let's pass the old Page and File objects to some hooks

### PR DESCRIPTION
**Pull request by pedroborges**
The affected hooks are:

- panel.page.update
- panel.page.sort
- panel.page.hide
- panel.file.replace
- panel.file.update
- panel.file.sort

### CODE REVIEW

File: app/src/panel/models/page.php [hunk 1]
```diff
@@ -284,7 +284,8 @@ public function canChangeTemplate() {
 
   public function move($uid) {
 
-    $old = clone($this);
+    // keep the old state of the page object
+    $old = clone $this;
```
Comment (pedroborges) [inline app/src/panel/models/page.php]: I only edited this to be consistent.

**Comment by jenstornell** [comment]
+1

**Comment by pedroborges** [comment]
Any chance of this making to the v2.3.0 release? Let me know if there's anything you want me to change.

**Comment by bastianallgeier** [comment]
I'd love to add it to 2.3.0 but I'm afraid we already have too much stuff in there and I'd rather give it a bit more attention before adding it. But it's at least going into 2.4.

**Comment by lukasbestle** [comment]
Thank you very much. This feature is now on the develop branch for Kirby 2.3.2.

**Comment by pedroborges** [comment]
Thank you!

**Pull request closed without merging**